package.json

```diff
@@ -31,14 +34,18 @@
   },
   "dependencies": {
     "cssmin": "^0.4.3",
+    "minimist": "^1.2.0",
     "node-sass": "^3.1.2"
   },
   "devDependencies": {
+    "cross-spawn": "^2.0.0",
+    "eslint": "^1.9.0",
     "mocha": "2.2.5",
     "proxyquire": "1.6.0",
     "sinon": "1.15.4"
   },
   "scripts": {
-    "test": "NODE_ENV=test mocha"
+    "test": "NODE_ENV=test mocha",
+    "pretest": "eslint *.js bin/scss-to-json src test"
```

Comment:
I don't think the script should generate the JSON files before the test suite is run. I'd prefer that the tests shell out to scss-to-json so that if the script fails, it will fail inside of a test with helpful error messages instead of pretest failing.

For an example of how to shell out to a CLI in a test, you can look here: https://github.com/wealthfront/verify-engine/blob/master/test/src-tests/cli-test.js